app/frontend/src/views/form/Create.vue

```diff
@@ -70,5 +70,13 @@ export default {
         this.$refs.settingsForm.validate();
     },
   },
+  beforeRouteLeave(to, from, next) {
+    const answer = window.confirm('Do you really want to leave this page?');
+    if (answer) {
+      next();
+    } else {
+      next(false);
+    }
+  },
```

Comment:
Could do
``` js
(answer) ? next() : next(false);
```
Also as a further note, I wonder if it would be prudent down the line to also move the standard nav leave stuff into here instead of having it reside in the component?

Reply:
This... isn't supposed to be in this PR. I must have comitted accidentally. The nav leave stuff isn't working any more as well... I'm looking at all that right now.
This can merge in though but will be adjusted.

Reply:
The warning on leave when creating a form broke here
![image](https://user-images.githubusercontent.com/17445138/96935357-bf1e5800-1478-11eb-9c73-7f4f236c400e.png)